src/main/java/hudson/plugins/gradle/injection/BuildScanInjection.java

```diff
@@ -2,14 +2,33 @@
 
 import hudson.EnvVars;
 import hudson.model.Node;
+import hudson.model.labels.LabelAtom;
+
+import java.util.HashSet;
+import java.util.Set;
+import java.util.stream.Collectors;
 
 public interface BuildScanInjection {
 
-    default boolean isOn(EnvVars env) {
+    default boolean injectionEnabled(EnvVars env) {
```

Comment:
Do we need this method at all? It's always being used with an && together with `isInjectionEnabledForNode`. I think we can just make it part of `isInjectionEnabledForNode`.

Reply:
Done